runtime/src/drivers.rs

```diff
@@ -228,21 +228,29 @@ impl Drivers {
         Ok(())
     }
 
+    // Caliptra Name serialNumber fields are sha256 digests
+    pub fn compute_rt_alias_sn(&mut self) -> CaliptraResult<CryptoBuf> {
+        let key = self.persistent_data.get().fht.rt_dice_pub_key.to_der();
+
+        let rt_digest = self.sha256.digest(&key)?;
```

Comment:
should we skip hashing the 0x04 der encoding byte?

Reply:
FMC hashes it and we have to do the same thing as FMC. Anyway, hashing the uncompressed format of the public key is a reasonable standard to follow

Reply:
SGTM